compiler/z/codegen/ControlFlowEvaluator.cpp

```diff
@@ -338,21 +338,29 @@ xmaxxminHelper(TR::Node* node, TR::CodeGenerator* cg, TR::InstOpCode::Mnemonic c
    generateS390LabelInstruction(cg, TR::InstOpCode::label, node, cFlowRegionStart);
    cFlowRegionStart->setStartInternalControlFlow();
 
+   //Checking common Condition Code and branching to cFlowRegionEnd
```

Comment:
I do not see any relevance of this comment. It is checking for the condition passed into this common utility function. Pls remove it.